modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb

```diff
@@ -14,14 +14,43 @@ def metadata
       {
         'veteranFirstName' => @data.dig('veteran', 'full_name', 'first'),
         'veteranLastName' => @data.dig('veteran', 'full_name', 'last'),
-        'fileNumber' => @data.dig('veteran', 'va_claim_number').presence || @data.dig('veteran', 'ssn'),
+        'fileNumber' => @data.dig('veteran', 'va_claim_number').presence || @data.dig('veteran', 'ssn_or_tin'),
         'zipCode' => @data.dig('veteran', 'address', 'postal_code') || '00000',
         'source' => 'VA Platform Digital Forms',
         'docType' => @data['form_number'],
         'businessLine' => 'CMP'
       }
     end
 
-    def track_user_identity; end
+    def handle_attachments(file_path)
+      attachments = get_attachments
+      if attachments.count.positive?
```

Comment:
Could probably get rid of `attachments = get_attachments` and just put `get_attachments.count.positive?`